clients/drcachesim/tracer/tracer.cpp

```diff
@@ -152,6 +154,16 @@ static volatile bool exited_process;
 static bool have_phys;
 static physaddr_t physaddr;
 
+/**
```

Comment:
This should be a regular comment: it's not docs to some user interface

Reply:
Done